### DISCUSSION TITLE
WSA 2305 and 2306 corrupting data?

### DISCUSSION THREAD

**Discussion by Almighty188** [LSPosed/MagiskOnWSALocal#622] [Q&A]
It seems with the last several versions of WSA 2305 and 2306, it will work fine when installed but then later, all apps will say WSA is starting which does start the WSA process but after 10 minutes or so, the app starting window disappears and the WSA process in services appears to have stopped.  The only way I have found to fix this is to first delete %LOCALAPPDATA%\Packages\MicrosoftCorporationII.WindowsSubsystemForAndroid_8wekyb3d8bbwe\LocalCache\userdata.vhdx before building and installing WSA again and ofcourse it will work for awhile and then I have to repeat the fix again.  Anyone else having this problem as it could also be an app causing it.   

This Windows app called WSATools allows backing up the WSA apps and data:
https://apps.microsoft.com/store/detail/wsatools-apk-installer-and-more/9N4P75DXL6FG?hl=en-us&gl=us&rtc=1

**Comment by krali27** [comment]
I had a similar issue a few weeks ago, my apps that had been working fine just stopped working and I couldn't get the subsystem to boot. I didn't do a huge amount of troubleshooting, after I updated it to the latest version, things snapped back into working again. (I think I went from a 2304.xx install to 2305.40000.5.0)

Haven't had an issue since but I'll try to keep a closer eye on it if it happens again.
